custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -54,6 +55,22 @@ def __init__(self, runtime):
         self.verbose = runtime.options.verbose
         self.run_mode = RunMode(runtime.options.subparser_name)
         self.raw_arguments = sys.argv
+        self.target_type = None
+
+        if self.run_mode != RunMode.NEW:
+            # if target type and class labels are not provided assume it is regression
+            if self.options.target_type is None:
+                if self.options.negative_class_label is None:
+                    self.options.target_type = TargetType.REGRESSION.value
```

Comment:
It would be good to add a log.info here since this is going to be "not the standard behavior" soon

Reply:
Also maybe adding some sort of a deprecation message that say that providing an explicit target type might be required in the future...

Reply:
I'll check a bit more if it is already better to make --target-type required

Reply:
Made --target-type mandatory for inference